gate/internal/repository/system_token_repository.go

```diff
@@ -162,6 +155,19 @@ func (systemTokenRepository systemTokenRepository) Update(systemToken domain.Sys
 	return nil
 }
 
+func (systemTokenRepository systemTokenRepository) UpdateRevoke(systemToken domain.SystemToken) error {
+	res := systemTokenRepository.db.
+		Table("system_tokens").
+		Where("id = ?", systemToken.ID).
+		UpdateColumns(models.SystemToken{Revoked: systemToken.Revoked, RevokedAt: systemToken.RevokedAt})
+
+	if res.Error != nil {
+		return handleSystemTokenError("Update system token failed", "SystemTokenRepository.UpdateRevoke.Updates", res.Error, logging.InternalError)
```

Comment:
```suggestion
		return handleSystemTokenError("Update system token failed", "SystemTokenRepository.UpdateRevoke.Update", res.Error, logging.InternalError)
```

* if the suggestion be applied to the method name, we need to remember to update here.